DatabaseAndReporting/Integration/01_ManageQueries.md

```diff
@@ -0,0 +1,140 @@
+[!!Manage the query categories](./02_ManageQueryCategories.md)
+[!!Manage the user rights](./05_ManageUserRights.md)
+[!!Execute the managed queries](../Operation/01_ExecuteManagedQueries.md)
+[!!User interface Queries](../UserInterface/01a_Queries.md)
+
+# Manage the queries
+
+Queries can be predefined to determine the data that may be accessed by specific user groups. Instead of entering and executing a query directly in the MySQL console, the user can simply select the query to be executed from the list of queries. 
```

Comment:
```suggestion
Queries can be predefined to determine the data that may be accessed by specific user groups. Instead of entering and executing a query directly in the MySQL console, the users can simply select the query to be executed from the list of queries. 
```